### DISCUSSION TITLE
Prevent closing by outside click

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#449] [Q&A] [answered]
I'm trying to create a Fancybox html inline content. Like in the example https://fancyapps.com/fancybox/plugins/html/  (press button 'Launch Dialog')
Which options should I use 
1) to prevent closing by click outside the content?
2) to prevent background scrolling by wheel?

**Comment by fancyapps** [comment, marked as answer]
Hi,

1) Sorry, I forgot to document several options. So, you simply use `backdropClick` option - https://fancyapps.com/fancybox/api/options/#backdropclick

2) I guess you refer to this issue - https://github.com/fancyapps/ui/issues/450 - that will be fixed in the next minor release.
